### DISCUSSION TITLE
Is there a way to use Form native submit to POST all RHF registered MUI TextInput fields not just native <input> ones?

### DISCUSSION THREAD

**Discussion by soulbeing** [react-hook-form/react-hook-form#8991] [Q&A]
I saw this and tried https://github.com/react-hook-form/react-hook-form/discussions/8864. But in the posted payload, it doesn't have any of the RHF registered MUI TextInput fields, it does have a few native <input> fields though... Other than that, it doesn't handle the response from the Post either.

Essentially, if my form is defined this way.
```
   <form
        action={actionUrl}
        method="post"
        onSubmit={onSubmit}
        noValidate
      >
```
When I click submit button, code in onSubmit will run, and after that, the form's default submit code will run(which uses action and method) and fires the form data.

But if I define my form like 
```
<form
        action={actionUrl}
        method="post"
        onSubmit={handleSubmit(onSubmit)}
        noValidate
      >
```
When I click submit button, code in onSubmit will run, and after that, nothing happens!!!  Instead of doing axios or fetch inside obSubmit(which will very likely introducing CORS issue), I want to use the form's default submit feature and at the same time make sure all fields data are submitted not just native <input> ones.

Any suggestions?

**Comment by Moshyfawn** [comment]
Not with the current design.

`handleSubmit` calls `.preventDefault` internally
